autotx/cli.py

```diff
@@ -92,7 +92,9 @@ def run(prompt: str | None, non_interactive: bool, verbose: bool, logs: str | No
         print(f"Smart account deployed: {manager.address}")
         
         if not is_safe_deployed:
-            send_native(dev_account, manager.address, 10, web3)
+            # XDAI or MATIC doesn't have the same value as ETH, so we need to fill more
+            amount_to_fill = 3000 if network_info.chain_id in [ChainId.POLYGON, ChainId.GNOSIS] else 10
```

Comment:
Can we move this logic to `fill_dev_account_with_erc20` and maybe rename it to `fill_dev_account_with_tokens`?